README.md

```diff
@@ -134,6 +135,19 @@ Separate each step of the task to an individual task. For example a "generate ic
 
 [↑ back to Table of Contents](#table-of-contents)
 
+## Use standard script names
+
+### Why?
+
+Makes it easier to any developer if the names of the tasks are consistent between projects.
+
+### How?
+
+Use names like `build`, `test`, `start` and `watch`.
```

Comment:
If we think there should be standardised script names, we should state what the meaning of each of them has. For example: `watch` is typically used to check for file changes and run another command like `build` or `test` on the changed files. (could do better, needs some discussion :) )